apps/prairielearn/elements/pl-string-input/pl-string-input.py

```diff
@@ -13,6 +15,17 @@ class DisplayType(Enum):
     BLOCK = "block"
 
 
+SPACE_HINT_DICT: dict[tuple[bool, bool], str] = {
```

Comment:
What is the advantage of defining this as a constant? I find this significantly harder to read than a series of inline `if (A, B):`... statements.

Reply:
Avoids having a large block of if statements + constant strings in the body of the function. IMO having these values baked into the function body makes it longer and harder to read if this isn't the part of the code you're specifically interested in.